e2e_tests/tests/per_provider/normal_tests/import_key.rs

```diff
@@ -406,5 +416,5 @@ fn import_ecc_private_key() {
         return;
```

Comment:
Can you also execute this test on the Mbed Crypto provider 😄? That way we will be sure that both the formats for the public key and key pair are correct 👌

Reply:
Done. And it looks like Mbed Crypto uses just the raw private key octet string from the whole structure.